code/modules/reagents/chemistry/recipes/others.dm

```diff
@@ -86,6 +88,126 @@
+	id = "crayongreen"
+	required_reagents = list("lye" =10, "greencrayonpowder" = 10)
+	required_temp = 374
+	mob_react = FALSE
+	
+/datum/chemical_reaction/crayonificationg/on_reaction(datum/reagents/holder, created_volume)
+	var/location = get_turf(holder.my_atom)
+	for(var/i = 1, i <= created_volume, i++)
+		new /obj/item/toy/crayon/green(location)
+	
+/datum/chemical_reaction/crayonificationb
+	name = "Blue Crayonification"
+	id = "crayonblue"
+	required_reagents = list("lye" =10, "bluecrayonpowder" = 10)
+	required_temp = 374
+	mob_react = FALSE
+	
+/datum/chemical_reaction/crayonificationb/on_reaction(datum/reagents/holder, created_volume)
+	var/location = get_turf(holder.my_atom)
+	for(var/i = 1, i <= created_volume, i++)
+		new /obj/item/toy/crayon/blue(location)
+	
+/datum/chemical_reaction/crayonificationp
+	name = "Purple Crayonification"
+	id = "crayonpurple"
+	required_reagents = list("lye" =10, "purplecrayonpowder" = 10)
+	required_temp = 374
+	mob_react = FALSE
+	
+/datum/chemical_reaction/crayonificationp/on_reaction(datum/reagents/holder, created_volume)
+	var/location = get_turf(holder.my_atom)
+	for(var/i = 1, i <= created_volume, i++)
+		new /obj/item/toy/crayon/purple(location)
+	
+/datum/chemical_reaction/crayonificationbla
+	name = "Black Crayonification"
+	id = "crayonblack"
+	required_reagents = list("lye" =10, "blackcrayonpowder" = 10)
+	required_temp = 374
+	mob_react = FALSE
+	
+/datum/chemical_reaction/crayonificationbla/on_reaction(datum/reagents/holder, created_volume)
+	var/location = get_turf(holder.my_atom)
+	for(var/i = 1, i <= created_volume, i++)
+		new /obj/item/toy/crayon/black(location)
+	
+/datum/chemical_reaction/crayonificationwhi
+	name = "White Crayonification"
+	id = "crayonwhite"
+	required_reagents = list("lye" =10, "whitecrayonpowder" = 10)
+	required_temp = 374
+	mob_react = FALSE
+	
+/datum/chemical_reaction/crayonificationwhi/on_reaction(datum/reagents/holder, created_volume)
+	var/location = get_turf(holder.my_atom)
+	for(var/i = 1, i <= created_volume, i++)
+		new /obj/item/toy/crayon/white(location)
+	
+/datum/chemical_reaction/crayonificationinv
+	name = "Mime Crayonification"
+	id = "crayonwinv"
+	required_reagents = list("lye" =10, "whitecrayonpowder" = 10, "blackcrayonpowder =10)
+	required_temp = 374
+	mob_react = FALSE
+	
+/datum/chemical_reaction/crayonificationinv/on_reaction(datum/reagents/holder, created_volume)
+	var/location = get_turf(holder.my_atom)
+	for(var/i = 1, i <= created_volume, i++)
+		new /obj/item/toy/crayon/mime(location)
+	
+/datum/chemical_reaction/crayonificationrai
+	name = "rainbow Crayonification"
+	id = "crayonwrai"
+	required_reagents = list("lye" =10, "colorful_reagent" = 10)
+	required_temp = 374
+	mob_react = FALSE
+	
+/datum/chemical_reaction/crayonificationrai/on_reaction(datum/reagents/holder, created_volume)
+	var/location = get_turf(holder.my_atom)
+	for(var/i = 1, i <= created_volume, i++)
+		new /obj/item/toy/crayon/rainbow(location)
```

Comment:
Ripo chef. Nutriment is useless now. Just eat rainbow crayon forever.

Reply:
Also Is there any reason this section is blue whereas other lines above are color coded?(Ex:/datum/ is always in orange)
Its a useless question I'm not any less curious knowing that.

Reply:
Rainbow will be more harderl to make in part 2